upload-cloud-storage/tests/upload-helper.test.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import 'mocha';
+import * as sinon from 'sinon';
+import { UploadHelper } from '../src/upload-helper';
+import { mockUploadFile } from './mocks/mockUploadFile';
+import { Storage } from '@google-cloud/storage';
+
+describe('Upload Helper', function() {
+  //stub for uploadFile
+  beforeEach(function() {
+    sinon.stub(UploadHelper.prototype, 'uploadFile').callsFake(mockUploadFile);
+  });
+
+  afterEach(function() {
+    sinon.restore();
+  });
+
+  it('uploads a single file', async function() {
```

Comment:
I'm not sure I understand the point of these tests, they just seem to be testing the mockUploadFile and not any functionality within UploadHelper.

Reply:
My testing methodology was flawed here. 
After some reading I realized I should actually be testing original UploadHelper with the GCS lib mocked out and I should be asserting that calls made to the GCS lib has expected args. My latest commit has this new approach, PTAL and let me know if this makes sense.